sources/packages/web/src/services/http/StudentApi.ts

```diff
@@ -24,7 +24,7 @@ export class StudentApi extends HttpBaseClient {
    * @param studentProfile information needed to create the user.
    */
   async createStudent(studentProfile: CreateStudentAPIInDTO): Promise<void> {
-    await this.postCall(this.addClientRoot("students"), studentProfile);
+    await this.postCall(this.addClientRoot("student"), studentProfile);
```

Comment:
Just to every body be on the same page, this was what @dheepak-aot mention about having the const.
We would be creating a const like below and reuing among all app calls. Dheepak, correct me if I am wrong.
```ts
const API_CONTROLLER =  "student";
```

```ts
await this.postCall(this.addClientRoot(API_CONTROLLER), studentProfile);
return this.postCall(this.addClientRoot(`${API_CONTROLLER}/search`), payload);
````

Reply:
maybe I can just make the change in this PR so that we don't forget about it.